3-data-sharing-exploration/README.md

```diff
@@ -257,6 +257,21 @@ az account set -s <SUBSCRIPTION_ID>
 chmod +x ./deploy.sh # NOTE: you might need to use sudo if you don't have enough permissions
 set -a && source ./src/.env && bash ./deploy.sh 
 ```
+
+NOTE:
```

Comment:
I think to maintain the structure, I would add this to the "Issues and Workarounds" section and in the Note we could write, if you are running in MacOS M1 please check the Issues and Workarounds section if you run into an error.

Reply:
Oh yes, makes sense. Will move it down there